etor/src/test/groovy/gov/hhs/cdc/trustedintermediary/etor/metadata/PartnerMetadataOrchestratorTest.groovy

```diff
@@ -222,6 +222,21 @@ class PartnerMetadataOrchestratorTest extends Specification {
         1 * mockPartnerMetadataStorage.readMetadata(receivedSubmissionId) >> Optional.of(metadata)
     }
 
+    def "getMetadata retrieves metadata successfully when receiver is present and sentSubmissionId is missing"() {
+        given:
+        String receivedSubmissionId = "receivedSubmissionId"
+        def bearerToken = "token"
+        def metadata = new PartnerMetadata(receivedSubmissionId, null, "sender", "receiver", Instant.now(), "hash")
```

Comment:
What is the probability of the sentSubmissionId being null?

Reply:
It's possible if `sendOrder` fails in `SendOrderUseCase.convertAndSend`. But it's not possible for `sentSubmissionId` to be null and `receiver` to be non null given `sentSubmissionId` is needed to get the `receiver`

Reply:
Can you change this test to be `getMetadata retrieves metadata successfully when sentSubmissionId and receiver are missing`, which will be more realistic scenario?